system_test/helpers/aest_docker_api.erl

```diff
@@ -111,6 +112,13 @@ inspect(ID) ->
     {ok, 200, Info} = docker_get([containers, ID, json]),
     Info.
 
+container_logs(ID) ->
+    Query = #{<<"stderr">> => <<"true">>, <<"stdout">> => <<"true">>},
+    {ok, 200, _RespHeaders, ClientRef} =
+        docker_get_raw([containers, ID, logs], Query),
```

Comment:
In my PR I added an option to `docker_get`, `docker_post` and `docker_delete` to specify the result type. If set to `raw` it would do the same... I think so at least...

Reply:
Useful - used. Thanks!